applications/cm-service/values.yaml

```diff
@@ -2,8 +2,28 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-# -- Number of web deployment pods to start
-replicaCount: 1
+config:
+
+  # -- Whether to echo SQLAlchemy generated SQL to the log
+  databaseEcho: false
+
+  # -- Logging level
+  logLevel: "INFO"
+
+  # -- Logging profile (`production` for JSON, `development` for
+  # human-friendly)
+  logProfile: "production"
+
+  outputVolume:
+    # -- If specified, name of storage class requested in service output area PVC
+    storageClassName: null
+    # -- Minimum storage requested in service output area PVC
+    storage: 1Gi
+    # -- If specified, sub-path within bound PV to be mounted at service output area
+    subPath: null
+
+  # -- URL path prefix
+  pathPrefix: "/cm-service/v1"
```

Comment:
I think we're going to want the v1 to be internal to the container so that the GUI can live at /cm-service/.

Reply:
Agreed. I will coordinate with Eman to rotate the paths on a subsequent ticket.